material/plugins/blog/plugin.py

```diff
@@ -370,7 +370,7 @@ def _is_excluded(self, post: Post):
         # and must be explicitly enabled by the author.
         if not isinstance(post.config.draft, bool):
             if self.config.draft_if_future_date:
-                return post.config.date.created > datetime.now()
+                return post.config.date.created > datetime.now().replace(tzinfo=timezone.utc)
```

Comment:
Does `datetime.now(timezone.utc)`maybe also work? I've seen taht somewhere, and it would be more concise.

Reply:
Yes! Updated.